docs/fireblocks.md

```diff
@@ -0,0 +1,92 @@
+# Fireblocks integration
```

Comment:
Since we are not installing the fireblocks package, I think we should say somewhere here the API version that we are currently using, if it exists.

Reply:
Done